python/tank/deploy/tank_commands/setup_project_params.py

```diff
@@ -1042,10 +1047,10 @@ def _process_config(self, config_uri):
                 # either a folder or zip file!
                 if config_uri.endswith(".zip"):
                     self._log.info("Hang on, unzipping configuration...")
-                    return (self._process_config_zip(config_uri), "local")
+                    return (self._process_config_zip(config_uri), "zip")
                 else:
                     self._log.info("Hang on, loading configuration...")
-                    return (self._process_config_dir(config_uri), "local")
+                    return (self._process_config_dir(config_uri), self._PROJECT_CONFIG)
```

Comment:
Minor: I think we should skip the constant here. My opinion is that it seems little bit odd that we have the following lines:

```
return (self._process_config_git(config_uri), "git")
return (self._process_config_zip(config_uri), "zip")
return (self._process_config_dir(config_uri), self._PROJECT_CONFIG)
return (self._process_config_app_store(config_uri), "app_store")
```

I think it's totally fine to just have "project" as a string here. We could of course have constants for all of them too - but maybe that's overkill? Sorry if this is a minor detail.